### ISSUE TITLE
Content and IA Consult from Beneficiary Travel Team (BTSSS Travel Pay)

### ISSUE THREAD

**Issue by carafrissell1** [department-of-veterans-affairs/va.gov-team#73678]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Beneficiary Travel Team
- **OCTO product owner:** Kay Lawyer
- **Product name:** Travel pay experience 
- **Product manager:**  Lauren Ernest (Cara Frissell best POC for this work)
- **Slack channel:** https://join.slack.com/share/enQtNjQ4MjU0NTg4OTAxMi1mOTQyNmQ4ZWMyZjgwNzQzNTFkODk0NjFiMDgyY2VjZmM4YTc2NDlmOTNjNTRmNmIyNWRmOTM5NmE1NjJhODMy
- **Dedicated content writer on your team (if you have one):** do not have one 
- **Dedicated a11y specialist on your team (if you have one):** do not have one 
 
## About your initiative

The Beneficiary Travel team thinks about the travel reimbursement process as an multi-channel experience.  From submission to payment, this is a process that our users experience on a multiple platforms.  Our teams goal is to simplify that experience.  

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else:  consult for research study

### What's the nature of your initiative and desired outcomes?

We are planning our first study to see where information about a travel claim status could live on VA.gov.  We would like feedback on the IA structures from @kristinoletmuskat  and feedback on content labeling to test from @laurwill .   

`Tell us briefly about your work and the outcomes you’re aiming to achieve.`

For the purpose of this consult, alignment on the treejack structure to ensure that we have accurately captured the extant structure and for content that we have considered all relevant labels to test. 

## Collaboration timeframe
A touchpoint or some feedback within the next 2 days would be helpful to meeting our aspirational timeline.  

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?

We are early in our exploratory process planning a study with users, we have not gone through any of the Collaboration cycle.  A touchpoint or some feedback within the next 2 days is necessary to meeting our aspirational timeline.  

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle. NOTE: this work is likely to lead to additional work that will through collab cycle in the future, but has not to date

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.

- [Research Plan Draft](https://github.com/department-of-veterans-affairs/va.gov-team/tree/fc843c4e16a574bd11be53e1db0855f4429539ba/products/health-care/beneficiary-travel/research/2024-01-Status%20Visibility%20IA%20Study) 
-  [ZenHub ticket](https://app.zenhub.com/workspaces/beneficiary-travel-btsss-65147e21930cd900223d8e64/issues/gh/department-of-veterans-affairs/va.gov-team/72295) 

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by carafrissell1** [comment]
Tree for feedback: 1.16
Older version of tree: 1.10
[Baseline1.10.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/13957366/Baseline1.10.xlsx)
[Baseline1.16.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/13957367/Baseline1.16.xlsx)

**Comment by laurwill** [comment]
Hey @carafrissell1 , great chatting yesterday — I looked at the research guide a bit closer, and want to amend my "use travel in all the labels" suggestion. 

If we want to find out if Veterans think of travel claims as different than other claims vs the same as other claims they'd find in the CST, then it might make sense to have an option that doesn't specify "travel." Does that make sense? I'm not an expert in tree testing, so I'll leave that up to you! Just something I thought of after we synced.
